src/Umbraco.Web.Common/ModelsBuilder/NoopModelsBuilderDashboardProvider.cs

```diff
@@ -2,5 +2,5 @@ namespace Umbraco.Cms.Web.Common.ModelsBuilder;
 
 public class NoopModelsBuilderDashboardProvider : IModelsBuilderDashboardProvider
 {
-    public string GetUrl() => string.Empty;
+    public string? GetUrl() => null;
```

Comment:
Breaking change 🛑  as per our announcement repo, we can only change nullability in major versions

Reply:
Hmm, the interface defines the return type as nullable though: https://github.com/umbraco/Umbraco-CMS/blob/c576bbea03875088685655bda20bf6bae8649207/src/Umbraco.Web.Common/ModelsBuilder/IModelsBuilderDashboardProvider.cs#L3-L6

So should we just keep the return type of the concrete implementation non-nullable, but return null (which seems weird, but actually does what you expect when dealing with the interface):

```suggestion
    public string GetUrl() => null!;
```

Or completely revert this change back to:

```suggestion
    public string GetUrl() => string.Empty;
```